www/css/ios-overrides.css

```diff
@@ -73,3 +73,20 @@
 .ios-hide{
   display: none !important;
 }
+
+.user-content {
+  background: linear-gradient(to bottom,#eb7125 50%, #efeff4 51%);
```

Comment:
Love this solution. So simple yet powerful!

Reply:
Haxx